### ISSUE TITLE
Faker fails in version 1.9.0

### ISSUE THREAD

**Issue by Gennon** [jormaechea/open-api-mocker#55]
I make use of the docker image for the mock api. 

I have this schema:

```
"schemas": {
      "User": {
        "type": "object",
        "properties": {
          "name": {
            "type": "string",
            "x-faker": "{{name.firstName}} {{name.lastName}}"
          }
        }
     }
  }
```
I get the following error using the 1.9.0 version:

```
Failed to generate fake result using {{name.firstName}} {{name.lastName}} schema. Falling back to primitive type.
```

However I get the correct result when I make use of the 1.8.0 version.

**Comment by jormaechea** [comment]
@Gennon, I just tried that schema and worked just fine. Which locale are you using? Maybe it's an issue with faker locale..

**Comment by Gennon** [comment]
@jormaechea thanks for looking into this! 🙌🏻

As I wrote in the beginning I am using the [Docker image](https://hub.docker.com/layers/jormaechea/open-api-mocker/1.9.0/images/sha256-15a2738312f59f63bb83a498cb947aecb6f3ccddf45bb158a922214837883c16?context=explore) and the locale is based on that. 

```
{ 
  locale: 'en',
  numberingSystem: 'latn',
  calendar: 'gregory',
  timeZone: 'UTC',
  year: 'numeric',
  month: 'numeric',
  day: 'numeric' 
}
```
I also tried the new 1.10.0 Docker Image and it has the same problem. 
From the diffs in the changes from 1.8.0 to 1.9.0 it is hard to see any reason why this should be a problem. And as far as I know there is no new version of faker from 1.8.0 to 1.9.0, but I might have missed something. 
Do you get the same issue if you make use of the docker image?

**Comment by jormaechea** [comment]
This was an issue with locales without the country modifier, such as `en` in the docker release.

Fix released in `v1.11.0` 🚀
